src/command_modules/azure-cli-network/azure/cli/command_modules/network/_params.py

```diff
@@ -339,9 +351,15 @@ def completer(prefix, action, parsed_args, **kwargs): # pylint: disable=unused-a
 register_cli_argument('network vpn-gateway create', 'sku', choices=get_enum_choices(sku))
 register_cli_argument('network vpn-gateway create', 'vpn_type', choices=get_enum_choices(vpnType))
 register_folded_cli_argument('network vpn-gateway create', 'public_ip_address', 'Microsoft.Network/publicIPAddresses', default_value_flag='existingId', allow_none=False, required=True)
+register_cli_argument('network vpn-gateway', 'cert_name', help='Root certificate name', options_list=('--name', '-n'))
+register_cli_argument('network vpn-gateway', 'gateway_name', help='Virtual network gateway name')
+register_cli_argument('network vpn-gateway root-cert create', 'public_cert_data', help='Base64 contents of the root certificate file or file path.', validator=load_cert_file)
```

Comment:
>Base64 contents of the root certificate file or file path. [](start = 88, length = 58)

From the validator, it looks like you are loading a cert file. Thus this should be aliased to something like (--cert-path) and the help text changed as appropriate. This is what I did for app-gateway so that it appears correct for CLI users as well as anyone that might be using the smart create (where they really would pass in the data because that's what the template requires)